lib/ocpp/v201/database_handler.cpp

```diff
@@ -41,9 +41,18 @@ void DatabaseHandler::sql_init() {
         throw std::runtime_error("Database access error");
     }
 
-    this->inintialize_enum_tables();
-}
+    // process any interrupted transactions before initializing
+    process_interrupted_transactions();
```

Comment:
I know you have to do this to fix the issue I created with this init table but I don't like that fact that we have to do this init here. I will think of a better solution to handle the `inintialize_enum_tables` so we don't have to do this step.

Reply:
What you could do here is do a simple `SELECT` statement of the `TRANSACTIONS` table without any criteria. If you get a `SQLITE_ROW` back you know there are transactions. If you get `SQLITE_DONE` back immediately you'd know there are no transactions.